container/Dockerfile

```diff
@@ -0,0 +1,19 @@
+#!BuildTag: suse/scc-hypervisor-collector:15.3
+FROM suse/sle15:15.3
+
+#USER wwwrun
```

Comment:
nit: we can remove these commented lines

Reply:
Yeah I was leaving them in to remind me to decide if we need them, but we don't need them ;-)